MessageBird/Client.cs

```diff
@@ -155,15 +155,80 @@ public void DeleteCallFlow(string id)
         /// <param name="id">The unique ID which was returned upon creation of a call flow.</param>
         /// <param name="callFlow"></param>
         /// <returns></returns>
-        public CallFlowResponse UpdateCallFlow(string id, CallFlow callFlow)
+        public VoiceResponse<CallFlow> UpdateCallFlow(string id, CallFlow callFlow)
         {
             ParameterValidator.IsNotNullOrWhiteSpace(callFlow.Title, "title");
             ParameterValidator.IsNotNull(callFlow.Steps, "steps");
 
             var callFlows = new CallFlows(new CallFlow { Id = id, Title = callFlow.Title, Steps = callFlow.Steps, Record = callFlow.Record });
             var result = restClient.Update(callFlows);
 
-            return (CallFlowResponse)result.Object;
+            return (VoiceResponse<CallFlow>)result.Object;
+        }
+
+
+        /// <summary>
+        /// Creating a call
+        /// </summary>
+        /// <param name="request"></param>
+        /// <returns></returns>
+        public VoiceResponse<Call> CreateCall(Call request)
+        {
+            ParameterValidator.IsNotNullOrWhiteSpace(request.Source, "source");
+            ParameterValidator.IsNotNullOrWhiteSpace(request.Destination, "destination");
+            ParameterValidator.IsNotNull(request.CallFlow, "callFlow");
+
+            var callResource = new Calls(request);
+            var result = restClient.Create(callResource);
+
+
+            return (VoiceResponse<Call>)result.Object;
+        }
+
+
+        /// <summary>
+        /// This request retrieves a listing of all calls.
+        /// If successful, this request returns an object with a data property, which is an array that has 0 or more recording objects.
+        /// </summary>
+        /// <param name="limit">Set how many records will return from the server</param>
+        /// <param name="offset">Identify the starting point to return rows from a result</param>
+        /// <returns>If successful, this request will return an object with a data, _links and pagination properties.</returns>
+        public CallList ListCalls(int limit = 20, int offset = 0)
+        {
+            var resource = new CallLists();
+
+            var list = (CallList)resource.Object;
+            list.Limit = limit;
+            list.Offset = offset;
+
+            restClient.Retrieve(resource);
+
+            return (CallList)resource.Object;
```

Comment:
Is this method working how you expect? Looks like the Retrieve method returns something that you don't use